.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Summary
+
+Summary of PR.
+
+### Related Issues
+
+- Resolves #
+
+### Status
+
+- [ ] Ready for merge
```

Comment:
Seems like there should me more states here... 

What is the status if the the submitter does not consider the PR "Ready for Merge"? 

One or more of these?
```
[ ] Requesting Feedback
[ ] Requesting Review
[ ] Requesting Approval
```

Reply:
I would argue that by saying they consider the work complete/ready to merge, the user is basically putting the ball in our court.  All three of those are true if it's considered ready to merge.

Reply:
then why are they submitting at PR at all?

Reply:
Sometimes you might submit one while working it to utilize travis